File: solutions/1421-find-numbers-with-even-number-of-digits/solution.java
class Solution {
    public int findNumbers(int[] nums) {
        int even=0;
        for(int i:nums){
            String s = Integer.toString(i);
            int len = s.length();
            if(len%2 == 0){
                even++;
            }
        }
        return even;
    }
}

File: solutions/1421-find-numbers-with-even-number-of-digits/solution.py
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        total = 0
        for i in nums:
            if i ==0:
                total+=1
            else:
                count = 0
                temp = i
                while temp>0:
                    count +=1
                    temp = temp//10
            if count%2 == 0:
                total+=1
        return total
        
